searx/engines/9gag.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+# pylint: disable=C0103
```

Comment:
FYI: We prefer to use the named option: `invalid-name` instead of `C0103` .. so it is easier to see what has been switched off.